iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergStorageHandler.java

```diff
@@ -722,8 +767,8 @@ public boolean canComputeQueryUsingStats(Partish partish) {
     if (hmsTable.getMetaTable() != null) {
       return false;
     }
-    if (!getStatsSource().equals(HiveMetaHook.ICEBERG) &&
-        StatsUtils.areBasicStatsUptoDateForQueryAnswering(partish.getTable(), partish.getPartParameters())) {
+    if (!getStatsSource().equals(HiveMetaHook.ICEBERG) && StatsSetupConst.areBasicStatsUptoDate(
+          partish.getPartParameters())) {
```

Comment:
nit: identation

Reply:
iceberg modules have auto checkstyle validations including indentation checks, so it should be ok